Tests/IntegrationTests/NavigatorConsistency.cpp

```diff
@@ -38,6 +38,10 @@ auto tGeometry = cGeometry();
 const double Bz = 2_T;
 auto bField = std::make_shared<ConstantBField>(Vector3{0, 0, Bz});
 
+Acts::Logging::Level logLevel = Acts::Logging::DEBUG;
```

Comment:
won't this result in `Debug` messages being printed right now?

Reply:
I leveled I bit. The consistency might be quite cluttered with the `DEBUG`, but I think the other test might be useful as `INFO`.